### PR TITLE
disable syslog and fix compile at windows

### LINKED ISSUE #14: Should the programm compile at windows?
Currently, this project can't be compiled at windows cause no `syslog` module available at this operating system. https://github.com/golang/go/blob/master/src/log/syslog/syslog.go#L5

Now time the easiest way is comment `syslog` import and usage. As options, it is available to make `syslog` disabled for windows with compiler directives as I think.

### PR DESCRIPTION
**Pull request by vblz**
Closes #14

### CODE REVIEW

File: app/main.go [hunk 1]
```diff
@@ -4,15 +4,14 @@ import (
...
 	"os"
 	"os/signal"
 	"syscall"
 
 	dockerclient "github.com/fsouza/go-dockerclient"
 	log "github.com/go-pkgz/lgr"
-	flags "github.com/jessevdk/go-flags"
-	lumberjack "gopkg.in/natefinch/lumberjack.v2"
+	"github.com/jessevdk/go-flags"
+	"gopkg.in/natefinch/lumberjack.v2"
```
Comment (vblz) [inline app/main.go]: Aliases was automatically deleted by goland as redundant

**Comment by umputun** [comment]
To me `main_*` seems confusing, maybe call it `syslog_*` ?

**Comment by vblz** [comment]
An idea was separate all platform specific code into two files. For this time it is only syslog related code, but it can be more in future. But I can rename if needed.

**Comment by umputun** [comment]
I really hope we won't need any other platform specific builds, even one is too many. Pls rename

**Comment by vblz** [comment]
done

**Comment by umputun** [comment]
thx. FYI - it will take forever to pass travis build -https://www.traviscistatus.com/incidents/my5wm56npf7q

**Pull request merged**